src/screens/PageHeader.js

```diff
@@ -394,7 +394,7 @@ export const PageHeaderItem = ({ name, path }) => (
     <PageHeader
       title="Page Title"
       subtitle="A subtitle for the page."
-      parent={<Anchor label="Parent Page" />}
+      parent={<Anchor as="span" label="Parent Page" />}
```

Comment:
Why is this added?

Reply:
The browser complains about `<a><a /></a>`, since the PageHeaderItem is rendered inside an anchor. This prevents that console error message.